src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -322,4 +320,5 @@
           authorization-fn
           whitelist-path
           (get-in ca-settings [:access-control :certificate-status]))
+        (i18n/locale-negotiator)
```

Comment:
nit: doesn't need to be changed, but just FYI: usually when you are writing a threading macro and there is a single-arg fn call, you won't see it wrapped with parens.  It doesn't hurt anything to wrap it with parens, just slightly less common syntax.